Source/Core/DiscIO/VolumeDirectory.cpp

```diff
@@ -353,9 +353,10 @@ void CVolumeDirectory::BuildFST()
   m_fst_data.clear();
 
   File::FSTEntry rootEntry = File::ScanDirectoryTree(m_root_directory, true);
-  u32 name_table_size = ComputeNameSize(rootEntry);
+  u32 name_table_size = Common::AlignUp(ComputeNameSize(rootEntry), 1ull << m_address_shift);
```

Comment:
I think the final name_offset should be rounded up before the assert otherwise the bytes at the end of the FST won't get zero'd. Game would probably handle it fine without, but we can't be 100% sure that this won't affect something.